workspace/data-sources/data.pages_403.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
```

Comment:
Pumping the 403 and 404 from the page section is not a good idea. Please remove this.